Britannia/localization/english/BTWK_buildings_l_english.yml

```diff
@@ -26,6 +26,16 @@
  ###############
  # BASIC MINES #
  ###############
+
+ building_btwk_mines_01:0 "Small Mining Community"
+ building_btwk_mines_01_desc:0 "Settlers have established a small community around a mineral rich area, intent to extract its valuable resources. This is mostly achieved by searching for small deposits above ground, like in bogs or streams, with minor excavation work being held to acquire the ore underground."
+ building_btwk_mines_02:0 "Surface Excavations"
+ building_btwk_mines_02_desc:0 "With the exhaustion of naturally occurring nodules on the surface, greater efforts have been spent to clear the land and expose the rich minerals bedded beneath. This has lead to the construction of a small mine shaft."
+ building_btwk_mines_03:0 "Support Structures"
+ building_btwk_mines_03_desc:0 "Before the shaft can be expanded further, measures must be put in place to prevent its future in-collapse, such as lining our tunnels regularly with wood and stone, and ensuring that the groundwater is regularly drained out." 
+ building_btwk_mines_04:0 "Deeper Shafts"
+ building_btwk_mines_04_desc:0 "To access the minerals deeper underground, we must employ the practice of #italic shaft sinking#! - vertically (or near-vertically) deepening a mine shaft, to gain better access to deeply bedded minerals. The excavation work here was one of the most dangerous jobs in the medieval world, and as such was usually only tasked to slaves and criminals."
```

Comment:
Not totally sure we want to mention the word "medieval" tbh

Reply:
Wasn't sure myself but I thought I'd leave it in and see what you thought. Will have a look at modifying it